variables.tf

```diff
@@ -56,3 +56,9 @@ variable "labels" {
   default     = {}
   type        = map(string)
 }
+
+variable "ssl_mode" {
+  description = "Specify how SSL connection should be enforced in DB connections"
+  default     = "ALLOW_UNENCRYPTED_AND_ENCRYPTED"
```

Comment:
I can see this is backwards compatible with the old behaviour.

I know nothing about this terraform module.  I don't know what uses it and why.  But.... `ALLOW_UNENCRYPTED_AND_ENCRYPTED` does not sound like a great default.  Should we consider breaking backwards compatibility with the old behaviour, and instead enforce ssl by default?

Reply:
I am not sure neither tbh. As you said, I just tried to preserve the backward compatibility.

@jbeemster What is you opinion here ?